src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/IService1.cs

```diff
@@ -44,4 +48,49 @@ public string StringValue
             set { stringValue = value; }
         }
     }
+
+    [ServiceContract (CallbackContract=typeof(IDuplexCallback))]
+    public interface IDuplexService
+    {
+        //[OperationContract(IsOneWay = true)]
+        [OperationContract]
```

Comment:
Nit: not required for merging, but commented out code should be removed where possible.  If it is a placeholder for some TODO work, we generally open an issue to track it.